google-cloud-firestore/src/test/java/com/google/cloud/firestore/QueryTest.java

```diff
@@ -194,13 +181,6 @@ public void limitToLastReversesResults() throws Exception {
 
   @Test
   public void limitToLastRequiresAtLeastOneOrderingConstraint() throws Exception {
-    doAnswer(queryResponse())
```

Comment:
Was it intentional to remove the `doAnswer(...)...` block from this test and the test below?